static/api/v0/openapi.yaml

```diff
@@ -1749,6 +1749,107 @@ paths:
               examples:
                 Internal Error:
                   $ref: '#/paths/~1v2~1users/get/responses/500/content/application~1json/examples/Internal%20Error'
+  '/v2/users/{user_id}/groups':
+    get:
+      tags:
+        - Users
+      operationId: GetUserGroups
+      summary: Retrieve an existing user groups
+      description: |
+        Get a list of groups to which a user belongs.
+      security:
+        - BearerAuth:
+            - 'users:read'
+            - 'groups:read'
+      parameters:
+        - $ref: '#/components/parameters/user_id'
+        - $ref: '#/components/parameters/page_size'
+        - $ref: '#/components/parameters/offset'
```

Comment:
We call this offset but in all other places we call it skip. I wonder if those are equivalent with a different name and if so we probably want to make those equivalent.

Reply:
Fixed this in the 2 locations it existed, we now use `skip`

Reply:
This was fixed for `/v2/passkeys:` and '/v2/users/{user_id}/groups'.